app/routes/product-details.$productSlug/route.module.scss

```diff
@@ -35,6 +35,13 @@
     line-height: 1.5;
 }
 
+.productOptions {
+    display: flex;
```

Comment:
why `flex`? To avoid styles like `.myClass:not(:last-child)` ? What are other reasons?

Reply:
Just the simplest way to add gap between elements.